packages/dx-react-chart-demos/src/demo-sources/interaction/single-multiple-selection.jsxt

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Chart,
+  BarSeries,
+  ArgumentAxis,
+  ValueAxis,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+
+import {
+  Scale, EventTracker, SelectionState,
+} from '@devexpress/dx-react-chart';
+
+const data = [
+  { year: '1950', population: 2.525 },
+  { year: '1960', population: 3.018 },
+  { year: '1970', population: 3.682 },
+  { year: '1980', population: 4.440 },
+  { year: '1990', population: 5.310 },
+  { year: '2000', population: 6.127 },
+  { year: '2010', population: 6.930 },
+];
+
+const compare = (
+  { series, point }, { series: targetSeries, point: targetPoint },
+) => series === targetSeries && point === targetPoint;
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data,
+      selection: [],
+    };
+
+    this.click = ({ targets, event }) => {
+      const target = targets[0];
+      if (target) {
+        this.setState(({ selection }) => {
+          if (event.ctrlKey) {
+            const filtered = selection.filter(el => !compare(el, target));
+            return {
+              selection: filtered.length < selection.length ? filtered : [...filtered, target],
+            };
+          }
+          return {
+            selection: selection[0] && compare(selection[0], target) ? [] : [target],
```

Comment:
Let's say there are 3 selected items by this moment. If one of them is clicked, selection is reset. If any of others - only that item is selected. Inconsistency.

When item is clicked all other items selection is reset and this item's selection is toggled.